### ISSUE TITLE
[BUG] Fail to install in Python 3.7 environment

### ISSUE THREAD

**Issue by josephwinston** [rapidsai/cudf#661]
**Describe the bug**
Installation fails on Ubuntu 18.04 with conda 4.5.12 running python 3.7.1

**Steps/Code to reproduce bug**
`$ conda install -c numba -c conda-forge -c nvidia -c rapidsai -c defaults cudf=0.4.0`

```
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - cudf=0.4.0
  - defaults/linux-64::mkl_random==1.0.2=py37hd81dba3_0
Use "conda info <package>" to see the dependencies for each package.
```

**Expected behavior**
Install cudf 

**Environment details (please complete the following information):**
 - Environment location: Cloud (azure)
 - Method of cuDF install: conda
 

**Additional context**
None

**Comment by kkraus14** [comment]
@josephwinston We don't support Python 3.7 quite yet, could you try a python 3.6 environment and see if that works for you?

**Comment by josephwinston** [comment]
Sure.  Also, anything I can do to help with 3.7 adoption?

**Comment by kkraus14** [comment]
If you'd like to try installing from source in a 3.7 environment and reporting back any issues with the unit tests that would be extremely helpful.

I believe the rest is related to CI pipeline changes that we are working to expose and control via source control management.

**Comment by josephwinston** [comment]
I'll do the source install and report back.  You can close this issue if you want.

**Comment by kkraus14** [comment]
This is a good issue / feature request and we can use this issue to discuss Python 3.7 support, so I'd like to keep this open. Thanks for opening this!

**Comment by mike-wendt** [comment]
#668 Just added Python 3.7 support and will be in v0.5 release by EOW

I'll keep this open for you to confirm it works once the release is out

**Comment by josephwinston** [comment]
Excellent news.

**Comment by kkraus14** [comment]
0.5 is released (with a 0.5.1 hotfix following shortly) with Python 3.7 support.
